Regressions/scripts/propeller/propeller_test.py

```diff
@@ -0,0 +1,401 @@
+# propeller_test.py
```

Comment:
Scrub the SUAVE out of this

Reply:
Done. The only thing here is that Battery_Propeller is still called from legacy SUAVE, as it doesn't seem to exist currently in RCAIDE. Assuming this is something @mclarke2  will have in his PR, and I can update this later if so

Reply:
Yeah make an issue associated with this and tag Matthew so he completes it with his PR